daemon/src/document.rs

```diff
@@ -199,10 +199,11 @@ impl Document {
     }
 
     pub fn files(&self) -> Vec<String> {
-        let file_map = self
-            .top_level_map_obj("files")
-            .expect("Failed to get files store from document");
-        self.doc.keys(file_map).collect()
+        if let Ok(file_map) = self.top_level_map_obj("files") {
+            self.doc.keys(file_map).collect()
+        } else {
+            vec![]
```

Comment:
I think it's fair to respond with the empty vector for a getter? That makes it possible to iterate over `files` here independent of whether a file exists or not:

https://github.com/ethersync/ethersync/blob/64cf875ca82f481e380237bd00e3e5954e668683/daemon/src/daemon.rs#L674

Reply:
Indeed!